.ci/install_kata_kernel.sh

```diff
@@ -30,6 +30,8 @@ kernel_repo_dir="${GOPATH}/src/${kernel_repo}"
 kernel_arch="$(arch)"
 readonly tmp_dir="$(mktemp -d -t install-kata-XXXXXXXXXXX)"
 packaged_kernel="kata-linux-container"
+#Experimental kernel support. Pull from virtio-fs GitLab instead of kernel.org
+experimental_kernel="false"
```

Comment:
`experimental_kernel="${experimental_kernel:-false}"` so that the value can be changed using envars?

Reply:
@chavafg changes applied